tools/tune/src/src_tiny_int16.m

```diff
@@ -41,7 +41,8 @@
 cfg.ctype = 'int16';
 cfg.profile = 'tiny';
 cfg.quality = 0.40;
+cfg.thdn = -90;
 cfg.speed = 1;
-cfg.gain = -1; % Make gain -1 dB
+cfg.gain = 0;
```

Comment:
What impact we see here with new setting ?

Reply:
Same, goes to other commit.